packages/fxa-settings/src/pages/Signin/Signin/index.tsx

```diff
@@ -0,0 +1,248 @@
+            <FtlMsg id="signin-header">
+              <h1 className="card-header">Sign in</h1>
+            </FtlMsg>
+            <>
+              {isPocketClient && (
+                <FtlMsg id="signin-subheader-with-logo">
+                  <span className="card-subheader">
+                    Continue to{' '}
+                    <span>
+                      <PocketLogo
+                        className="inline"
+                        aria-label={localizedPocketAriaLabel}
+                      />
+                    </span>
+                  </span>
+                </FtlMsg>
+              )}
+              {!isPocketClient && serviceLogo && ServiceLogo && (
+                <FtlMsg id="signin-subheader-with-logo">
+                  <span className="card-subheader">
+                    Continue to{' '}
+                    <span>
+                      <ServiceLogo
+                        className="inline"
+                        aria-label={localizedLogoAriaLabel}
+                      />
+                    </span>
+                  </span>
+                </FtlMsg>
+              )}
+              {!isPocketClient && !serviceLogo && (
+                <FtlMsg
+                  id="signin-subheader-without-logo"
+                  vars={{ serviceName }}
+                >
+                  <span className="card-subheader">
+                    Continue to {serviceName}
+                  </span>
+                </FtlMsg>
+              )}
+            </>
+          </>
+        )}
+      </div>
+      <section>
+        {/* Alerts and success messages originally went here */}
+        <div className="mt-9">
+          <div className="avatar-wrapper mx-auto h-24 w-24 mobileLandscape:h-40 mobileLandscape:w-40"></div>
+          <div className="my-5 text-base break-all">{email}</div>
+        </div>
+        <form noValidate {...{ onSubmit }}>
+          <input type="email" className="email hidden" value={email} disabled />
+
+          {isPasswordNeeded && (
+            <InputPassword
+              anchorStart={true}
+              className="mb-2 text-start"
+              label={localizedPasswordFormLabel}
+              hasErrors={error.length > 0}
+              errorText={error}
+              tooltipPosition="bottom"
+              required={true}
+              autoFocus={true}
+              onChange={(e) => setPassword(e.currentTarget.value)}
+            />
+          )}
+          {/* This non-fulfilled input tricks the browser, when trying to
+              sign in with the wrong password, into not showing the doorhanger.
+           */}
+          <input className="hidden" required />
+
+          <div className="flex">
+            <FtlMsg id="signin-button">
+              <button className="cta-primary cta-xl" type="submit">
+                Sign in
+              </button>
+            </FtlMsg>
+          </div>
+        </form>
+        <div id="tos-pp" className="text-grey-500 my-5 text-xs">
```

Comment:
I don't believe we need this `tos-pp` ID. I think we're going to want to update our functional test selectors to use `data-testid` or some other selector instead whenever we get there...